docs/user/production-considerations/task-manager-troubleshooting.asciidoc

```diff
@@ -0,0 +1,710 @@
+[role="xpack"]
+[[task-manager-troubleshooting]]
+=== Task Manager troubleshooting
+
+++++
+<titleabbrev>Troubleshooting</titleabbrev>
+++++
+
+Task Manager is used by a wide range of services in {kib}, such as <<alerting-production-considerations, Alerting>>, Reporting and Telemetry.
+Unexpected behavior in these services might be a downstream issue originating in Task Manager.
+
+This page describes how to resolve common problems you might encounter with Task Manager.
+If your problem isn’t described here, please review open issues in the following GitHub repositories:
+
+* https://github.com/elastic/kibana/issues[kibana] (https://github.com/elastic/kibana/issues?q=is%3Aopen+is%3Aissue+label%3A%22Feature%3ATask+Manager%22[Task Manager issues])
+
+Have a question? Contact us in the https://discuss.elastic.co/[discuss forum].
+
+[float]
+[[task-manager-health-scheduled-tasks-small-schedule-interval-run-late]]
+==== Tasks with small schedule intervals run late
+
+*Symptom*:
+
+Tasks are scheduled to run every 2 seconds but they seem to be running too late
+
+*Resolution*:
+
+Task Manager polls for tasks at the cadence specified by the <<task-manager-settings,`xpack.task_manager.poll_interval`>> setting, which is 3 seconds by default. This means that a Task could run late if it uses a schedule which is smaller than this setting.
+
+This can be addressed by adjusting the <<task-manager-settings,`xpack.task_manager.poll_interval`>> setting, though this will add additional load to both {kib} and {es} instances in the cluster, as they will peform more queries.
+
+[float]
+[[task-manager-health-tasks-run-late]]
+==== Tasks Run Late
```

Comment:
I'd say the first one is a subset of the second, rather than a less common occurrence.
For that reason, the first one could be confused for the second... and I'd like to make it more likely they spot that sooner.
Otherwise they might go into deep investigation trying to find the reason their alert is late, when it was simply configured with too low an interval.